commit 时 flake8 报 E501 line too long

### 请提出你的问题 Please ask your question

pre-commit 和 commit 时都报 flake8 Failed，显示 E501 line too long.

```check for added large files..............................................Passed
check for merge conflicts................................................Passed
check for broken symlinks............................(no files to check)Skipped
detect private key.......................................................Passed
fix end of files.........................................................Passed
sort simple yaml files...............................(no files to check)Skipped
trim trailing whitespace.................................................Passed
CRLF end-lines remover...................................................Passed
Tabs remover (C++)...................................(no files to check)Skipped
Tabs remover (Python)....................................................Passed
copyright_checker........................................................Passed
black....................................................................Passed
isort....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

python/paddle/nn/initializer/Bilinear.py:63:80: E501 line too long (80 > 79 characters)
python/paddle/nn/initializer/Bilinear.py:64:80: E501 line too long (80 > 79 characters)
python/paddle/autograd/py_layer.py:31:80: E501 line too long (106 > 79 characters)
python/paddle/autograd/py_layer.py:36:80: E501 line too long (112 > 79 characters)
python/paddle/autograd/py_layer.py:56:80: E501 line too long (87 > 79 characters)
......```

--- a/.flake8
+++ b/.flake8
@@ -13,16 +13,26 @@ exclude =
     ./python/paddle/fluid/tests/unittests/npu/**,
     ./python/paddle/fluid/tests/unittests/mlu/**
 ignore =
-    E203, # Whitespace before ‘,’, ‘;’, or ‘:’, it is not compatible with black
-    E402, # Module level import not at top of file
-    E501, # Line too long (82 > 79 characters)
-    E721, # Do not compare types, use `isinstance()`
-    E722, # Do not use bare except, specify exception instead
-    E731, # Do not assign a lambda expression, use a def
-    E741, # Do not use variables named ‘l’, ‘O’, or ‘I’
-    F405, # `name` may be undefined, or defined from star imports: `module`
-    F841, # Local variable name is assigned to but never used
-    W503  # Line break before binary operator, it is not compatible with black
+    # Whitespace before ‘,’, ‘;’, or ‘:’, it is not compatible with black
+    E203,
+    # Module level import not at top of file
+    E402,
+    # Line too long (82 > 79 characters)
+    E501,
+    # Do not compare types, use `isinstance()`
+    E721,
+    # Do not use bare except, specify exception instead
+    E722,
+    # Do not assign a lambda expression, use a def
+    E731,
+    # Do not use variables named ‘l’, ‘O’, or ‘I’
+    E741,
+    # `name` may be undefined, or defined from star imports: `module`
+    F405,
+    # Local variable name is assigned to but never used
+    F841,
+    # Line break before binary operator, it is not compatible with black
+    W503
 per-file-ignores =
     # These files need tabs for testing.
     python/paddle/fluid/tests/unittests/dygraph_to_static/test_error.py:E101,W191

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -64,9 +64,10 @@ repos:
     hooks:
     -   id: isort
 -   repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
+    rev: 5.0.4
     hooks:
     -   id: flake8
+        args: ["--config=.flake8"]
 -   repo: https://github.com/PyCQA/autoflake
     rev: v1.7.7
     hooks:

